modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -270,6 +270,30 @@ export default {
         this.allPiecesSelection.isSelected = this.checked.length === this.allPiecesSelection.total ||
           (this.checked.length && this.maxReached());
       }
+    },
+    getDocModuleName(doc) {
+      // Don't assume the piece has the type of the module,
+      // this could be a virtual piece type such as "submitted-draft"
+      // that manages docs of many types
+      if (doc) {
+        return doc.slug.startsWith('/') ? '@apostrophecms/page' : doc.type;
+      }
+      return this.moduleName;
+    },
```

Comment:
This one was used in `AposDocsManager` moved here to be reused in the below method.